modules/blog/src/main/BlogTransform.scala

```diff
@@ -23,7 +23,7 @@ object BlogTransform:
       .maximumSize(64)
       .build[Int, Html]()
 
-    private val PreRegex = """<pre>markdown(.+)</pre>""".r
+    private val PreRegex = """<pre>markdown([^<]++)</pre>""".r
```

Comment:
Was a bit unsure about this one at first but I don't think the pre-tags can legitimately contain `<`? It should be escaped.